Frends.Files.Move/Frends.Files.Move.Tests/WindowsServer/docker-compose.yml

```
@@ -0,0 +1,60 @@
+      retries: 5
+      start_period: 60s
+    ports:
+      - "1445:445"
+
+volumes:
+  fileserver_data:
+    driver: local
+
+networks:
+  test-net:
+    driver: nat
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)